packages/components/src/components/Bar/styles/mixins.scss

```diff
@@ -0,0 +1,125 @@
+// ---------------------------------------------
+// BAR - MIXINS
+// ---------------------------------------------
+
+@use "./config";
+@use "../../../tokens";
+@use "../../../styles/utils";
+
+
+@mixin bar {
+  display: flex;
+  // TODO FIX wrapping problems 
+  // - FLEX OFF not shrinking but stacking with wider elements
+  // - FLEX ON not wrapping with bar break 
+  // flex-wrap: wrap; 
+  margin-bottom: tokens.space();
+}
+
+@mixin bar-layout($direction: "horizontal") {
+  @if $direction == "horizontal" {
+    flex-direction: row;
+    align-items: center;
+    margin-top: -(tokens.space("s"));
+
+    .bar__item {
+      margin-top: tokens.space("s");
+    }
+  } @else if $direction == "vertical" {
+    height: 100%;
+    flex-direction: column;
+    align-items: stretch;
+  }
+}
+
+@mixin bar-item($width: 0) {
+  flex: 0 0 if($width == 0, auto, $width);
+  min-width: 0;
+  align-items: center;
+
+  > :not([class*="mb\-"]):last-child {
+    margin-bottom: 0;
+  }
```

Comment:
`mb-X` ma important, reku netreba to tu takto specifikovat.

Reply:
nerozumiem :) čo tu netreba?

Reply:
oznacene pravidlo resetuje spodny margin len pre elementy, ktore nepouzivaju utilitu na vlastny spodny margin.
utilita na spodny margin ale pouziva `!important`, co je silnejsia specificita ako hore spomenuta, a teda nemusime robit `not` podmienku